include/mbedtls/platform.h

```diff
@@ -266,12 +272,12 @@ extern int (*mbedtls_nv_seed_read)( unsigned char *buf, size_t buf_len );
 extern int (*mbedtls_nv_seed_write)( unsigned char *buf, size_t buf_len );
 
 /**
- * \brief   Set your own seed file writing/reading functions
+ * \brief   This function sets your own seed file writing or reading functions.
```

Comment:
Suggestion: Follow the same wording as above, i.e. `This function allows configuring custom seed file writing and reading functions.`

Reply:
Fixed